repo/packages/N/nexclipper-agent/0/resource.json

```diff
@@ -1,18 +1,16 @@
 {
   "images": {
```

Comment:
We shouldn't delete from the universe. Could you leave the old package and just increase the min dcos version to 999 as done here: https://github.com/mesosphere/universe/blob/b5b257e7f92f7e647ae7e90993fdc880f466d7d1/repo/packages/D/dse/0/package.json#L6

Reply:
P.S. I believe that is also causing the one test failure below

Reply:
OK I'll leave the old version and convert current version's minDcosReleaseVersion to 999. Thank you

Reply:
convert the old  version to 999 ;)